components/index/VolunteerHow.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <hr class="c-how__divider">
+    <div class="c-how">
```

Comment:
You should be able to get rid of this div and move the class to the root div of the template. You can just remove the padding on `.c-how` and increase the top margin on `&__section`